src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -80,19 +80,20 @@ def run(self, revision):
         logger.info('Modified files', files=modified_files)
 
         # mach configure
-        logger.info('Mach configure ...')
+        logger.info('Mach configure…')
```

Comment:
i understand it is 2017 and utf-8 should be generally accepted and working in all programs. my experience shows that this is far from true.

logs will be consumed by sentry/papertrail/<any terminal we use> and it wouldn't be nice to have this failing just because we want to be fancy 😉 

I would rather us use ASCII characters in the code. I don't have strong objections against it, I just had with few tool bad experience when using utf-8 and would like to to avoid it. 

@jankeromnes do you feel strong about this and you want to continue using non ASCII chars?
@marco-c @La0 do you have any preference?